appinventor/lib/blockly/src/core/block.js

```diff
@@ -2157,4 +2157,77 @@ Blockly.Block.prototype.renderDown = function() {
   }
   // [lyn, 04/08/14] Because renderDown is recursive, doesn't make sense to track its time here.
 };
+//****************************CURRENTLY ADDED (CECE & ABHIJIT) 7/30/15****************************//
+/**
+ * Select this block.  Highlight it visually.
+ */
+Blockly.Block.prototype.searchHighlight = function() {
+  goog.asserts.assertObject(this.svg_, 'Block is not rendered.');
+  if (Blockly.searched) {
+    // Unselect any previously selected block.
+    Blockly.searched.unSearchHighlight();
+  }
+  Blockly.searched = this;
+  this.svg_.addSearchHighlight();
+  // this.svg_.getRootElement().setAttribute('stroke-width','1px');
```

Comment:
All the blockly core has been commented out so that it would be easier for Andrew to merge @josmas